packages/curve-ui-kit/src/shared/ui/CrvHubIcon.tsx

```diff
@@ -0,0 +1,14 @@
+import { createSvgIcon } from '@mui/material/utils'
```

Comment:
I think it would be nice to create a separate folder for icons

Reply:
I put them in `curve-ui-kit\src\shared\icons` for now, together with `DodoIcon.tsx`.